app/src/main/java/pl/droidevs/books/dao/InMemoryBookDao.java

```diff
@@ -61,6 +62,11 @@ private void addBook(String title, String author, String category, String imageU
         return booksLiveData;
     }
 
+    @Override
+    public LiveData<BookEntity> getBookById(int bookId) {
+        return null;
```

Comment:
Gdy jest już podpięta baza danych to ten mock jest już niepotrzebny. Ale jeśli już dodałeś taką metodę to na pewno nie powinna ona zwracać nulla.